Source/Core/Core/IOS/ES/TitleManagement.cpp

```diff
@@ -350,7 +350,7 @@ IPCCommandResult ES::ImportContentEnd(Context& context, const IOCtlVRequest& req
 
 ReturnCode ES::ImportTitleDone(Context& context)
 {
-  if (!context.title_import.tmd.IsValid())
+  if (!context.title_import.tmd.IsValid() || context.title_import.content_id != 0xFFFFFFFF)
```

Comment:
IMO, there should be a `context.title_import.valid` for checking if the import context is valid, instead of checking the TMD for validity in all functions other than ImportTitleInit; and something similar for content imports, as checking for a special value for the content ID is not quite as clear.

I have a WIP branch that merges the import/export context and makes checks more similar to the actual behaviour on console, but I thought it'd be best to leave that for another PR.